.github/workflows/Updater.yml

```diff
@@ -79,7 +79,7 @@ jobs:
         env:
           UPDATE_BRANCH_NAME: ${{ steps.set-variables.outputs.UPDATE_BRANCH_NAME }}
         run: |
-          ruby tools/update_ruby_gem_packages.rb
+          LD_LIBRARY_PATH=/usr/local/lib ruby tools/update_ruby_gem_packages.rb
```

Comment:
This shouldn't be necessary and not advisable for x86_64.

Reply:
This is a requirement to load ruby-libversion in the github actions (not inside the chromeos image) container. We already do this for python packages.

Reply:
Did we install ruby-libversion incorrectly then?  All libraries for x86_64 should go in a `lib64` directory.

Reply:
Yes.